### PR TITLE
unit test for sbt_renew

### PR DESCRIPTION
**Pull request by sczembor**
- add unit test for `registry.sbt_renew` method

### CODE REVIEW

File: contracts/registry/src/lib.rs [hunk 1]
```diff
@@ -398,7 +398,7 @@ mod tests {
     }
 
     const START: u64 = 10;
-    const MINT_DEPOSIT: Balance = 6 * MILI_NEAR;
+    const MINT_DEPOSIT: Balance = 6 * 1_000_000_000_000_000_000_000;
```
Comment (sczembor) [inline contracts/registry/src/lib.rs:401]:
```suggestion
    const MINT_DEPOSIT: Balance = 6 * MILI_NEAR;
```

File: contracts/registry/src/registry.rs [hunk 2]
```diff
@@ -205,7 +205,7 @@ impl SBTRegistry for Contract {
         let storage_start = env::storage_usage();
         let storage_deposit = env::attached_deposit();
         require!(
-            storage_deposit >= 6 * MILI_NEAR,
+            storage_deposit >= 6 * 1_000_000_000_000_000_000_000,
```
Comment (sczembor) [inline contracts/registry/src/registry.rs:208]:
```suggestion
            storage_deposit >= 6 * MILI_NEAR,
```

File: contracts/registry/src/lib.rs (hunk 1 above)
Comment (sczembor) [inline contracts/registry/src/lib.rs:401]:
```suggestion
    const MINT_DEPOSIT: Balance = 6 * MILI_NEAR;
```

File: contracts/registry/src/lib.rs [hunk 3]
```diff
@@ -769,4 +769,150 @@ mod tests {
...
+        assert_eq!(
+            ctr.sbt_tokens_by_owner(alice(), Some(issuer1()), None, None),
+            vec![(
+                issuer1(),
+                vec![
+                    mk_owned_token(1, m1_1.clone()),
+                    mk_owned_token(2, m2_1.clone())
+                ]
+            ),]
+        );
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply contracts/registry/src/lib.rs:791]: this is already tested, no need to duplicate

File: contracts/registry/src/lib.rs [hunk 4]
```diff
@@ -769,4 +769,150 @@ mod tests {
...
         assert_eq!(bs[1].1, 103, "alice must be first in the iterator");
     }
+
+    #[test]
+    fn registry_renew_one_issuer() {
+        let (_, mut ctr) = setup(&issuer1(), 3 * MINT_DEPOSIT);
+
+        // mint two tokens
+        let m1_1 = mk_metadata(1, Some(START + 10));
+        let m2_1 = mk_metadata(2, Some(START + 10));
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply contracts/registry/src/lib.rs:779]: just to have a bit more randomness

**Review by robert-zaremba — APPROVED** [review]
pre-approved
let's clean a bit the tests to reduce duplicated code

File: contracts/registry/src/lib.rs [hunk 5]
```diff
@@ -769,4 +769,150 @@ mod tests {
...
+        assert_eq!(
+            ctr.sbt_tokens_by_owner(alice(), Some(issuer1()), None, None),
+            vec![(
+                issuer1(),
+                vec![
+                    mk_owned_token(1, m1_1.clone()),
+                    mk_owned_token(2, m2_1.clone())
+                ]
+            ),]
+        );
```
[the remark this answers is not in the document]
Comment (sczembor) [reply contracts/registry/src/lib.rs:820]: DONE

**Pull request merged**